e2e/Makefile

```diff
@@ -85,7 +85,7 @@ notice:
 
 .PHONY: sync-integrations
 sync-integrations:
-	OP_LOG_LEVEL=${LOG_LEVEL} ./op sync integrations --delete
+	OP_LOG_LEVEL=${LOG_LEVEL} ./op sync integrations --delete --remote "elastic:7.7"
```

Comment:
This is the first version having the majority of the integrations the supported-versions.yml file.